src/components/error_boundary/error_boundary.tsx

```diff
@@ -1,23 +1,40 @@
 import React, { Component } from 'react';
+import { CommonProps } from '../common';
 import PropTypes from 'prop-types';
 
 import { EuiText } from '../text';
 
-export class EuiErrorBoundary extends Component {
+interface Error {
+  stack?: any;
+}
+
+interface EuiErrorBoundaryState {
+  hasError: boolean;
+  error?: Error;
+}
+
+export type EuiErrorBoundaryProps = EuiErrorBoundaryState & CommonProps & {};
```

Comment:
`EuiErrorBoundaryProps` shouldn't contain `EuiErrorBoundaryState`, as the error values aren't meant to be provided (this should allow you to revert the changes to the test file)

```suggestion
export type EuiErrorBoundaryProps = CommonProps & HTMLAttributes<HTMLDivElement>;
```

Reply:
Ok, removed from the Props interface, but then had to create the component as `export class EuiErrorBoundary extends Component<{}, EuiErrorBoundaryProps & ErrorState>` since accessing `this.state = errorState;` was giving `Type 'ErrorState' has no properties in common with type 'Readonly<EuiErrorBoundaryProps>'`.